build.gradle

```diff
@@ -32,6 +32,10 @@ ext {
     androidxExifVersion = '1.0.0'
     androidxDesignVersion = '1.0.0-rc01'
 
+    // rx
+    rxJavaVersion = '2.2.2'
```

Comment:
We should be able to use the latest version: https://github.com/ReactiveX/RxJava/releases

Reply:
Good catch, updated.